api/src/main/java/keywhiz/api/model/Group.java

```diff
@@ -59,7 +59,7 @@ public class Group {
   private final ImmutableMap<String, String> metadata;
 
   @JsonProperty
-  private final String owner;
+  private String owner;
```

Comment:
This was an oversight in the original model change: being able to mutate this property makes the DAO code much cleaner in that you don't have to recreate the same Group twice, once without an owner and then once with.